aws/resource_aws_ecs_service.go

```
@@ -353,9 +374,36 @@ func resourceAwsEcsServiceRead(d *schema.ResourceData, meta interface{}) error {
 		log.Printf("[ERR] Error setting placement_constraints for (%s): %s", d.Id(), err)
 	}
 
+	if err := d.Set("network_configuration", flattenEcsNetworkConfigration(service.NetworkConfiguration)); err != nil {
+		log.Printf("[ERR] Error setting network_configuration for (%s): %s", d.Id(), err)
```

Any reason why we should silently fail here instead of returning error?